docs/Networking.asciidoc

```diff
@@ -52,6 +52,102 @@ or Wicked way:
 </interface>
 ---------------
 
+## TAP with Open vSwitch
+
+The recomended way to configure the network for TAP devices is using Open vSwitch.
+There is a support service +os-autoinst-openvswitch.service+ which sets vlan number
+of Open vSwitch ports based on +NICVLAN+ variable - this separates the groups of
+tests from each other.
+Compared to VDE setup discussed later, Open vSwitch is more complicated to configure,
+but provides more robust and scalable network.
+
+Starting Open vSwitch and adding TAP devices:
+[source, bash]
+---------------
+
+# start openvswitch.service
+systemctl start openvswitch.service
+systemctl enable openvswitch.service
+
+#create bridge
+ovs-vsctl add-br br0
+
+# add tap devices, use vlan 999 by default, the vlan number is supposed to be changed when the vm starts
+ovs-vsctl add-port br0 tap0 tag=999
+ovs-vsctl add-port br0 tap1 tag=999
+ovs-vsctl add-port br0 tap2 tag=999
+ovs-vsctl add-port br0 tap3 tag=999
+ovs-vsctl add-port br0 tap4 tag=999
+
+---------------
+
+If the workers with TAP capability are spread across multiple hosts, the network must be connected.
```

Comment:
From this sentence I didn't get that the following is an example of such network connection.